### PR TITLE
Add several products to our package

### PR DESCRIPTION
**Pull request by johndelong**
This MR adds a few more products to our Package and renames our package to `MSLToolKit`.

Our package now contains, `MSLFoundation`, `MSLCoreData`, `MSLNetworking`, `MSLUIKit`, and `MSLSwiftUI` as products.

### CODE REVIEW

**Comment by johndelong** [comment]
Documentation should be added

File: Sources/MSLCoreData/Utilities/DataStorePublisher.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        super.init()
+        self.fetchedRequestController.delegate = self
+
+        self.fetchedRequestController.managedObjectContext.perform {
+            try? self.fetchedRequestController.performFetch()
+            self.emitObjects(for: self.fetchedRequestController)
+        }
+    }
+
+    public var publisher: AnyPublisher<[Model], Never> {
```
Comment (scottschmitz) [inline Sources/MSLCoreData/Utilities/DataStorePublisher.swift:42]: Can we please document these public variables/functions?

File: Sources/MSLUIKit/UIViewController+Transition.swift [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+///
+/// Example:
+///
+/// ```
+/// self.rootViewController.present(
+///    to: self.caseListViewController,
+///    withOptions: .init(style: .swipe, direction: .right)
+/// )
+/// ```
+public extension UIViewController {
```
Comment (scottschmitz) [inline Sources/MSLUIKit/UIViewController+Transition.swift:14]: This file starts out with a nice comment but then there just aren't any after that... could we add some more?

**Comment by scottschmitz** [comment]
We need to discuss what license we want to associate with this. I think my proposal would be MIT license.

File: Sources/MSLNetworking/NetworkStatusCode.swift [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+public enum HTTPStatusCode {
+    public static let ok = 200
```
Comment (scottschmitz) [inline Sources/MSLNetworking/NetworkStatusCode.swift:4]: OKAY

**Comment by scottschmitz** [comment]
I added the MIT license

**Pull request merged**